plenum/common/messages/client_request.py

```diff
@@ -75,6 +76,15 @@ class ClientTxnAuthorAgreementOperation(MessageValidator):
     )
 
 
+# TODO: Add more fields along with implementation
+class ClientGetTxnAuthorAgreementOperation(MessageValidator):
```

Comment:
I believe we need a specific validation that either `GET_TXN_AUTHOR_AGREEMENT_VERSION` or `GET_TXN_AUTHOR_AGREEMENT_DIGEST` can be specified, not both.

Reply:
Thanks for pointing at it